scripts/scil_validate_bids.py

```diff
@@ -251,9 +271,13 @@ def main():
 
         # Get the data for each run of DWIs
         for dwi in dwis:
-            data.append(get_data(nSub, dwi, t1s, associations, args.readout))
+            data.append(get_data(nSub, dwi, t1s, associations,
+                                 args.readout, args.clean))
+
+    if args.clean:
+        data = [d for d in data if not all(d.values())]
```

Comment:
Your condition is inverted no ? If I run tractoflow with penthera 3T taking PT001 and --clean neither subjects are in the json file.